cmd/policy/images.go

```diff
@@ -6,11 +6,15 @@ type ImagesCmd struct {
 	Remote    bool   `name:"remote" short:"r" help:"List policies from a remote registry."`
 	Server    string `name:"server" short:"s" help:"Registry server to connect to." default:"{{ .DefaultDomain }}"`
 	ShowEmpty bool   `name:"show-empty" short:"e" help:"Show policies with no images."`
+	Org       string `name:"organization" short:"o" help:"Show images for an organization."`
 }
 
 func (c *ImagesCmd) Run(g *Globals) error {
 	if c.Remote {
-		err := g.App.ImagesRemote(c.Server, c.ShowEmpty)
+		if c.Org == "" {
```

Comment:
might be fixed with the `required` struct tag.

Reply:
Cannot use required as the org flag is only needed when listing remote images